### PR TITLE
appsync emulator: adds dynamodbConfig option to tester's server creation

### PR DESCRIPTION
**Pull request by cbaron**
It was more difficult than expected to get java >= 8 on a CircleCI build instance

- [x] tester allows dynamodb config to be passed in to skip dynamodb emulator creation

### CODE REVIEW

**Comment by cbaron** [comment]
@lightsofapollo -- not sure what happened to travis.

**Comment by cbaron** [comment]
cc @pianomansam

**Comment by lightsofapollo** [comment]
@cbaron Is "use localstack" what we want or to pass options for a dynamodb emulator that overrides the default behavior of spinning up a process ?

File: packages/appsync-emulator-serverless/tester.js [hunk 1]
```diff
@@ -3,13 +3,26 @@ const { createSchema } = require('./schemaTest');
...
+  serverless,
+  schemaPath,
+  port = 0,
+  dynamodbConfig = null,
+} = {}) => {
+  let dynamodb;
+  let emulator;
+
+  if (dynamodbConfig) {
+    dynamodb = new DynamoDB(dynamodbConfig);
```
Comment (lightsofapollo) [inline packages/appsync-emulator-serverless/tester.js]: Nit: add quick comment about why we're not spinning up an emulator in this case.

File: packages/appsync-emulator-serverless/tester.js [hunk 2]
```diff
@@ -3,13 +3,26 @@ const { createSchema } = require('./schemaTest');
 const createServerCore = require('./serverCore');
 const testJWT = require('./testJWT');
 const dynamoEmulator = require('@conduitvc/dynamodb-emulator/client');
+const { DynamoDB } = require('aws-sdk');
```
Comment (lightsofapollo) [inline packages/appsync-emulator-serverless/tester.js]: nit: move this to line 18 since we only need the import in that specific case.

**Pull request merged**